apps/portal/app/lib/hooks/useDebounce.tsx

```diff
@@ -0,0 +1,12 @@
+import { useEffect, useState } from 'react'
```

Comment:
Great! If we decide we want to adopt using `lodash` in the future for our utils we can update to use lodash's debounce but this is great until then.